libs/web/shared/ui/src/lib/input/input.props.ts

```diff
@@ -0,0 +1,19 @@
+import type { BorderSize, BorderSizeProps, Height, SizeProps, Width } from '../props';
+
+export const ContainerVariants = ['primary', 'secondary', 'tertiary', 'error'] as const;
+export type ContainerVariant = (typeof ContainerVariants)[number];
+
+export interface TextInputProps extends BorderSizeProps, SizeProps {
+  rounded?: BorderSize;
+  width?: Width;
+  height?: Height;
```

Comment:
`width` and `height` property already extend from `SizeProps` so it safe to remove that

Reply:
Gotcha.